README.md

```diff
@@ -100,6 +100,70 @@ resources:
 Don't forget, you'll need to mount the directory containing the unix socket that
 the KMS server is listening on into the kube-apiserver.
 
+### Bootstrap during cluster creation (kops)
+To use encryption provider during cluster creation, you need to ensure that its running
+before starting kube-apiserver. For that you need to follow following high level steps.
```

Comment:
...to ~~follow~~ perform the following...

Reply:
done.